### DISCUSSION TITLE
友链页面可以添加菜单之类的右侧导航吗？

### DISCUSSION THREAD

**Discussion by zuoer96** [hugo-next/hugo-theme-next#116] [Q&A]
友链页面可以添加菜单之类的右侧导航吗？
如果多了，滚动要找好久
是否可以将title作为右侧的菜单，方便查看这个title下的友链

**Comment by elkan1788** [comment]
title 链接形式也解决不了你的需求，比如当某个 title 下面的友链很多的时候，虽然能快速定位，但是一样也是要滚动才能找到你想看的那个友链，可考虑给 title 增加个点击收缩/展开的设计，或者看看大家能否有更好的设计思路。
